client/src/components/Query/index.jsx

```diff
@@ -494,6 +502,7 @@ View
             key={operator.key}
             options={options}
             data={operator.data}
+            intl={intl}
```

Comment:
@claudia1296 It would be best practice to use the intl component as intended, as it works as a "higher-order" component, automatically passing down it's props once `injectIntl`~

See example here: https://github.com/cr-ste-justine/clin-frontend/blob/dev/client/src/components/Header.jsx